plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -100,11 +102,12 @@ object ScalafmtPlugin extends AutoPlugin {
       config: Path,
       log: Logger,
       writer: OutputStreamWriter,
-      resolvers: Seq[Resolver]
+      resolvers: Seq[Resolver],
+      isDetailedError: Boolean
```

Comment:
```suggestion
      detailedErrorEnabled: Boolean
```

more understandable name